Sources/CombineBluetooth/Models/CentralManager.swift

```diff
@@ -14,7 +14,14 @@ public protocol CentralManager: BluetoothManager {
     func connect(_ peripheral: BluetoothPeripheral) -> AnyPublisher<BluetoothPeripheral, BluetoothError>
     func connect(_ peripheral: BluetoothPeripheral, options: [String : Any]) -> AnyPublisher<BluetoothPeripheral, BluetoothError>
     func peripheral(for uuid: UUID) -> BluetoothPeripheral?
+    
     // TODO: Nice to have, but complicate to handle single delegate (subscribing again with different options should end prior observations, which can
     //       be a bit unexpected).
     // - (void)registerForConnectionEventsWithOptions:(nullable NSDictionary<CBConnectionEventMatchingOption, id> *)options NS_AVAILABLE_IOS(13_0);
 }
+
+public struct CentralManagerFactory {
+    public static func create(from centralManager: CBCentralManager) -> CentralManager {
+        CoreBluetoothCentralManager(centralManager: centralManager)
```

Comment:
If I am not missing a point, we can also get rid of this.
CoreBluetoothCentralManager(centralManager: CBCentralManager) exactly does the same.

Reply:
We don't want to expose `CoreBluetoothCentralManager` to the outside, otherwise the `CentralManager` protocol would be obsolete, so this creator is needed.

Reply:
No it would not be obsolete. 

CoreBluetoothCentralManager(fooCentralManager) as CentralManager will be valid since you conform CoreBluetoothCentralManager: CentralManager.

Reply:
I got what you mean. We were using 

```
    public var combine: CentralManager {
        CoreBluetoothCentralManager(centralManager: self)
    }
```

this previously, now we need to pass a CBCentralManager to initialize CentralManager. But since this is an extension of CBCentralManager, you can still use combine property to initialize CentralManager. E.g. 
```
let customCM = CustomCentralManager: CBCentralManager { }
 customCM.combine as CentralManager
```

you still don't need this factory stuff.

Reply:
Ah now I get what you mean, yeah makes sense, refactored it